pkg/schedule/schedulers/evict_slow_store.go

```diff
@@ -274,40 +258,64 @@ func (handler *evictSlowStoreHandler) updateConfig(w http.ResponseWriter, r *htt
 	if err := apiutil.ReadJSONRespondError(handler.rd, w, r.Body, &input); err != nil {
 		return
 	}
-	recoveryDurationGapFloat, ok := input["recovery-duration"].(float64)
-	if input["recovery-duration"] != nil && !ok {
+	recoveryDurationGapFloat, inputRecoveryDuration := input["recovery-duration"].(float64)
+	if input["recovery-duration"] != nil && !inputRecoveryDuration {
 		handler.rd.JSON(w, http.StatusInternalServerError, errors.New("invalid argument for 'recovery-duration'").Error())
 		return
 	}
 
-	batch := handler.config.getBatch()
-	batchFloat, ok := input["batch"].(float64)
-	if input["batch"] != nil && !ok {
+	batchFloat, inputBatch := input["batch"].(float64)
+	if input["batch"] != nil && !inputBatch {
 		handler.rd.JSON(w, http.StatusInternalServerError, errors.New("invalid argument for 'batch'").Error())
 		return
 	}
-	if ok {
+	if inputBatch {
 		if batchFloat < 1 || batchFloat > 10 {
 			handler.rd.JSON(w, http.StatusBadRequest, "batch is invalid, it should be in [1, 10]")
 			return
 		}
-		batch = (int)(batchFloat)
+	}
+
+	enableNetworkSlowStore, inputEnableNetworkSlowStore := input["enable-network-slow-store"].(bool)
+	if input["enable-network-slow-store"] != nil && !inputEnableNetworkSlowStore {
+		handler.rd.JSON(w, http.StatusInternalServerError, errors.New("invalid argument for 'enable-network-slow-store'").Error())
+		return
 	}
 
 	handler.config.Lock()
 	defer handler.config.Unlock()
+	if !inputRecoveryDuration {
```

Comment:
Why not initialize these configurations first and then update them only if the parse is successful?